aldryn_events/models.py

```diff
@@ -428,7 +454,15 @@ class UpcomingPluginItem(BaseEventPlugin):
     latest_entries = models.PositiveSmallIntegerField(
         verbose_name=_('latest entries'),
         default=5,
-        help_text=_('The number of latests events to be displayed.')
+        help_text=_('The number of latest events to be displayed.')
+    )
+
+    cache_duration = models.PositiveSmallIntegerField(
+        default=0,  # not the most sensible, but consistent with older versions
+        blank=False,
+        help_text=_(
+            "The maximum duration (in seconds) that this plugin's content "
+            "should be cached.")
```

Comment:
NOTE: When I attempted to use a ModelMixin for this package, Django kept trying to tell me that I couldn't use @python_2_unicode_compatible, because the class didn't declare __str__(), which, of course it does.